src/Compilers/CSharp/Test/Semantic/Semantics/GenericConstraintsTests.cs

```diff
@@ -2342,6 +2360,26 @@ .locals init (pinned T& V_0)
 }");
         }
 
+        [Fact]
+        public void UnmanagedConstraints_CtorAndValueTypeAreEmitted()
+        {
+            CompileAndVerify(@"
+using System.Linq;
+class Program
+{
+    public static void M<T>() where T: unmanaged
+    {
+    }
+
+    static void Main(string[] args)
+    {
+        var typeParam = typeof(Program).GetMethod(""M"").GetGenericArguments().First();
+        System.Console.WriteLine(typeParam.GenericParameterAttributes);
+    }
+}",
+    options: TestOptions.UnsafeReleaseExe, verify: Verification.Passes, expectedOutput: "NotNullableValueTypeConstraint, DefaultConstructorConstraint");
```

Comment:
I think `Verification.Passes` is the default. Consider removing.

Reply:
Considering that many uses of “unmanaged” result in unverifiable code, I think explicit Passes makes sense here.